040-SDK/103-update.mdx

```diff
@@ -152,6 +152,63 @@ user, _ := client.Upsert(context.TODO(), xata.UpsertRecordRequest{
 
 </TabbedCode>
 
+## Bulk update
+
+If you have multiple records to update, you can send them using the [`/transactions` endpoint](/docs/sdk/transaction). The Xata transactions API allows bulk updates where specified fields of records are modified. These updates require an ID and will fail if the IDs aren't found in the database. For more details, refer to the [transactions updates documentation](https://xata.io/docs/sdk/transaction#updates).
+
+Alternatively, you can send them in a single request via the [`/bulk` endpoint](/api-reference/db/db_branch_name/tables/table_name/bulk#bulk-insert-table-records). For example:
+
+<TabbedCode tabs={['TypeScript', 'Python', 'Go', 'SQL', 'JSON']}>
+
+```ts
+const updatedUsers = await xata.db.Users.bulkUpdate([
```

Comment:
@joan-ing there is no bulkUpdate endpoint (neither something in the works afaik).

Instead, the transactions endpoint can be used with the update operation.

Here's the equivalent call with the hypothetical bulkUpdate for this example:

```ts
const result = await xata.transactions.run([
  {
    update: {
      table: "Users",
      id: "rec_cmf56ugvjdkilkpcdeu0",
      fields: { email: "neo@matrix.com", name: "Neo" },
    },
  },
  {
    update: {
      table: "Users",
      id: "rec_cmf572veoo9ahn3r96p0",
      fields: { email: "trinity@matrix.com", name: "Trinity" },
    },
  },
]);
```

In the update docs page, there is areference pointing to the transactions page for the update operation: https://xata-docs-pr-224.vercel.app/docs/sdk/update#updating-multiple-records

we may want to enhance that section instead of adding a "Bulk update" one because it is kind of misleading since it's not the bulk endpoint that performs the update, but the transaction one (with whatever a transaction entails -> one operation failing means all operations in the request are rolled back)

Reply:
Thanks Kostas! I think we should havd content here as well as beef up the transactions multiple records section. As a user, I think they would first navigate to the update section to find how to update multiple records. Even though we do not have bulk endpoint, maybe we should include the bulk update/transaction code snippet in the update section and have some conceptual info in the transactions doc?

Reply:
I agree we can add examples in the update doc for how a bulk update looks like using a transaction (we should still point to the transactions doc as the source of truth / more operations supported there).